aws-sdk-v2/src/main/java/io/micronaut/aws/sdk/v2/service/s3/S3ConfigurationProperties.java

```diff
@@ -34,10 +36,28 @@ public class S3ConfigurationProperties extends AWSConfiguration {
     @ConfigurationBuilder(prefixes = {""}, excludes = {"profileFile", "applyMutation"})
     private Builder builder = S3Configuration.builder();
 
+    private URI endpointOverride;
+
     /**
      * @return The builder
      */
     public Builder getBuilder() {
         return builder;
     }
+
+    /**
+     * @return The endpoint with which the AWS SDK should communicate
+     * @since 3.6.2
+     */
+    public URI getEndpointOverride() {
+        return endpointOverride;
+    }
+
+    /**
+     * @param endpointOverride The endpoint with which the AWS SDK should communicate
```

Comment:
Improve the setter description. Say that default value is `null`. The setter description  is used for description here: https://micronaut-projects.github.io/micronaut-aws/latest/guide/configurationreference.html#io.micronaut.aws.sdk.v2.service.s3.S3ConfigurationProperties

Reply:
The description is taken verbatim from [the own AWS SDK](https://github.com/aws/aws-sdk-java-v2/blob/master/core/sdk-core/src/main/java/software/amazon/awssdk/core/client/builder/SdkClientBuilder.java#L57).

Also, are 3 `@Nullable` annotations not enough to indicate that this is null?

Reply:
Yes, they are if you open the class. However, if you are reading the docs you have no idea if it is optional unless we tell them. I added a commit. Now it looks like: 
<img width="1092" alt="Screenshot 2022-08-02 at 17 04 25" src="https://user-images.githubusercontent.com/864788/182407784-482e4742-67f1-48c7-810a-658a63b21b8e.png">